Description: add iceweasel to browser list, use Thunar instead of xftree as filemanager
Author: Yves-Alexis Perez <corsac@debian.org>
Forwarded: not-needed

diff --git a/scripts/xfbrowser4.in b/scripts/xfbrowser4.in
index a55f44a..77f81fb 100644
--- a/scripts/xfbrowser4.in
+++ b/scripts/xfbrowser4.in
@@ -2,7 +2,7 @@
 
 if [ "x$BROWSER" = "x" ]
 then
-  for b in exo-open @BROWSER_DEFAULT@ firefox epiphany galeon mozilla konqueror dillo
+  for b in exo-open @BROWSER_DEFAULT@ iceweasel firefox epiphany galeon mozilla konqueror dillo
   do
     if which "$b" > /dev/null 2>&1
     then
diff --git a/scripts/xfmountdev4 b/scripts/xfmountdev4
index 937db34..6ef97ef 100644
--- a/scripts/xfmountdev4
+++ b/scripts/xfmountdev4
@@ -38,7 +38,7 @@ fi
 # seems clearer to me -- Jasper
 if [ "x$FILEMAN" = "x" ]
 then
-  FILEMAN=xftree4
+  FILEMAN=thunar
 fi
 
 if [ "x$1" = "x" ]; then
